### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by sahiba0915**
Hello team, Kindly review my code for home page,
-Added styling
-Added images

Live Preview: https://stylevilla-homepage.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <!--Header-->
+    <header class="header">
+      <div class="header-1">
+        <a href="./index.html" class="logo">
+          <i class="fa-solid fa-bag-shopping"></i> StyleVilla
+        </a>
+        <form action="" class="search-form">
+          <input
+            type="search"
+            name=""
```
Comment (chanchal-panpaliya) [inline index.html]: name is empty

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        <div class="box">
+          <div class="image">
+            <img src="image/card1.jpg" alt="" />
+            <small class="small-txt">New Arrivals</small>
+          </div>
+        </div>
+
+        <div class="box">
+          <div class="image">
+            <img src="image/card2.jpg" alt="" />
```
Comment (chanchal-panpaliya) [inline index.html]: alt is empty

File: index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    <section class="feature" id="featured">
+      <h1 class="heading"><span> Limited edition </span></h1>
+
+      <div class="row">
+        <div class="image-container">
+          <div class="banner-img">
+            <img src="./image/hero.jpg" alt="" />
+          </div>
+        </div>
+      </div>
```
Comment (abhishek17551) [inline index.html]: can probably either add some text over the image or add multiple images like in a carousel. would make the page more visually appealing.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+  cursor: pointer;
+  font-weight: 500;
+}
+
+.btn:hover {
+  background: var(--pink);
+}
+
+.header .header-1 {
+  background: #fff;
```
Comment (chanchal-panpaliya) [inline style.css]: use var( )

**Review by abhishek17551 — APPROVED** [review]
recommend to add some text in 'alt' attribute of image tag.
rest all is fine.
UI looks clean and good.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+  }
+
+  .header .header-1 {
+    box-shadow: var(--box-shadow);
+    position: relative;
+  }
+
+  .header .header-1 .search-form {
+    position: absolute;
+    top: -115%;
```
Comment (chanchal-panpaliya) [inline style.css:377]: all non-negative values are valid percentage values above 100%. "-115%" values ,Is this correct?

**Review by chanchal-panpaliya — APPROVED** [review]
code is good to merge .

**Comment by sahiba0915** [comment]
Merging Home page

File: index.html (hunk 1 above)
Comment (sahiba0915) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (sahiba0915) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (sahiba0915) [reply index.html]: done

File: style.css (hunk 4 above)
Comment (sahiba0915) [reply style.css]: done

**Pull request merged**